runtime/browser/xwalk_runner.cc

```diff
@@ -183,6 +185,22 @@ void XWalkRunner::DisableRemoteDebugging() {
   remote_debugging_server_.reset();
 }
 
+component_updater::ComponentUpdateService* XWalkRunner::component_updater() {
+  if (!component_updater_.get()) {
+//    if (!content::BrowserThread::CurrentlyOn(BrowserThread::UI))
+//      return NULL;
+    scoped_refptr<update_client::Configurator> configurator =
+        component_updater::MakeXwalkComponentUpdaterConfigurator(
+            base::CommandLine::ForCurrentProcess(),
+            browser_context_->GetRequestContext());
+    // Creating the component updater does not do anything, components
+    // need to be registered and Start() needs to be called.
+    component_updater_.reset(component_updater::ComponentUpdateServiceFactory(
```

Comment:
<code>component_updater_</code> is a <code>scoped_ptr</code> pointer. I prefer to using <code>scoped_ptr</code>'s <code>reset</code> method. Do you any concern for this usage?